letsencrypt/cli.py

```diff
@@ -1112,6 +1158,13 @@ def __init__(self, args, plugins):
         self.parser._add_config_file_help = False  # pylint: disable=protected-access
         self.silent_parser = SilentParser(self.parser)
 
+        # This setting attempts to force all default values to None; it
```

Comment:
Incorrect comment. It doesn't force all default values to `None`, it forces them to falsy values.